pennylane/ops/op_math/pow.py

```diff
@@ -268,14 +268,15 @@ def has_decomposition(self):
             return False
         return True
 
-    def decomposition(self):
+    @staticmethod
+    def compute_decomposition(*params, wires, z, base):
```

Comment:
See conversation above. If we want to change how we are doing these types of functions in operator arithmetic, that should be an independent PR.